docs/sources/tasks/collect-prometheus-metrics.md

```diff
@@ -51,7 +51,7 @@ To configure a `prometheus.remote_write` component for metrics delivery, complet
 
    - _`<LABEL>`_: The label for the component, such as `default`.
      The label you use must be unique across all `prometheus.remote_write` components in the same configuration file.
-   - _`<PROMETHEUS_URL>`_ The full URL of the Prometheus-compatible endpoint where metrics will be sent, such as `https://prometheus-us-central1.grafana.net/api/prom/push`.
+   - _`<PROMETHEUS_URL>`_ The full URL of the Prometheus-compatible endpoint where metrics are sent, such as `https://prometheus-us-central1.grafana.net/api/v1/write` for Prometheus or `https://mimir-us-central1.grafana.net/api/prom/push/` for Mimir. The endpoint URL depends on the database you use.
```

Comment:
The Prometheus endpoint looks right, but for vanilla Mimir the endpoint looks to be `/api/v1/push` ([source](https://grafana.com/docs/mimir/latest/references/http-api/#remote-write)). I _think_ the endpoint that's here for Mimir is one used for Grafana Cloud specifically, not sure if we want to make that distinction.

Reply:
I'll update for vanilla Mimir. It's OK to add Cloud stuff, especially if it's in examples.... in this case generic is better.